networking/src/Mockable/Concurrent.hs

```diff
@@ -46,10 +46,11 @@ module Mockable.Concurrent (
 
   ) where
 
-import           Control.Exception (AsyncException (ThreadKilled), Exception)
+import           Control.Exception (AsyncException (..))
+import           Control.Exception.Safe hiding (throwTo)
+--import           Control.Exception (AsyncException (ThreadKilled), Exception)
```

Comment:
First, I see implicit imports, second, is this commented out line needed?

Reply:
this is oops